robot-core/src/main/java/org/obolibrary/robot/ReportOperation.java

```diff
@@ -944,13 +944,14 @@ private static String getQueryResultOrNull(QuerySolution qs, String var) {
         OWLEntity e = dataFactory.getOWLClass(ioHelper.createIRI(property));
         if (value != null) {
           IRI valIRI = ioHelper.createIRI(value);
-          if (valIRI != null) {
-            violation.addStatement(e, valIRI);
+          if (valIRI != null && !valIRI.toString().contains(" ")) {
```

Comment:
Seems like `!valIRI.toString().contains(" ")` should belong into ioHelper.createIRI()? Just a suggestion to push potential problems upstream.

Reply:
I think you're right about this. I'll change it so that `IOHelper.createIRI` returns `null` if the IRI contains a space.

Reply:
cool